src/common/utils/paths.ts

```diff
@@ -22,9 +22,9 @@
 import path from "path";
 import os from "os";
 
-function resolveTilde(filePath: string) {
+export function resolveTilde(filePath: string, home?: string) {
```

Comment:
I take home as optional param because on windows, `os.homedir()` returns `$USERPROFILE` instead of `$HOME`, which results in a path like: `C:\\Users\user` VS `/c/Users/user`.

Reply:
So I would just use `%USERPROFILE%` (for `~`) as that is guaranteed to exist and seems to be what powershell does. If it doesn't work for a user then they can just use `%HOME%` themselves.